src/components/layout/Tutorial/Tutorial.jsx

```diff
@@ -34,20 +33,15 @@ const Tutorial = ({ onFinish }) => {
   const hasPreviousStep = getStep(activeStepPosition - 1) !== undefined
   const goToStep = useCallback(newStepPosition => () => setActiveStepPosition(newStepPosition), [])
 
+  const activeStep = getStep(activeStepPosition)
+
   return (
     <div
       className="tutorial"
       data-testid="tutorial-container"
     >
       <section className="content-section">
-        {steps.map(step => (
-          <TutorialStep
-            isActive={activeStepPosition === step.position}
-            key={`tutorial-step-${step.position}`}
-          >
-            <step.component />
-          </TutorialStep>
-        ))}
+        <activeStep.component />
```

Comment:
```suggestion
        <activeStepComponent />
```
C'est une question de gout et de couleur, mais je trouve ça plus simple à lire